tests/tt_metal/tt_metal/test_compile_sets_kernel_binaries.cpp

```diff
@@ -191,15 +191,15 @@ int main(int argc, char **argv) {
                             JitBuildProcessorType::DATA_MOVEMENT,
                             0,
                             get_latest_kernel_binary_path(mask, riscv0_kernel));
-                        ll_api::memory brisc_binary = llrt::get_risc_binary(brisc_hex_path);
+                        ll_api::memory brisc_binary = llrt::get_risc_binary(brisc_hex_path, 0, true);
```

Comment:
from api standpoint I recommend to replace bool arg to an enum, as it makes semantics of this `true` clear.

Reply:
The integer constant should be an enum too, AFAICT it's an 'id'

Reply:
I'll change the bool.  The riscv ID is entrenched throughtout the code base w/ many different definitions.  It will get cleaned up when we start using the HAL to abstract this away.